package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -154,7 +154,7 @@
   #represents MSD vs t
   i=0
   for msd in MSD_analysis.timeseries:
-        print i +" "+ msd
+        print(i +" "+ msd)
```

Comment:
I did not correct then al, though. For a few of them, it would have slightly changed the output of the example. Making the example work the same for both version would have make it more complicated.